infra/examples-dev/aws-google-workspace/main.tf

```
@@ -85,6 +85,8 @@ module "psoxy" {
   salesforce_domain              = var.salesforce_domain
   jira_server_url                = var.jira_server_url
   jira_cloud_id                  = var.jira_cloud_id
+  github_organization            = var.github_organization
+  example_github_repository      = var.example_github_repository
```

we should prefix everything by connector, easier to group visually and find:
```
```
```suggestion
  github_organization            = var.github_organization
  github_example_repository      = var.example_github_repository
```
and move up `example_jira_issue_id` with the other jira's.
If they were as I said you wouldn't have splitted them :smile: